src/Makefile.am

```diff
@@ -588,13 +588,13 @@ clean-local:
 check-symbols: $(bin_PROGRAMS)
 if GLIBC_BACK_COMPAT
 	@echo "Checking glibc back compat..."
-	$(AM_V_at) READELF=$(READELF) CPPFILT=$(CPPFILT) $(top_srcdir)/contrib/devtools/symbol-check.py < $(bin_PROGRAMS)
+	$(AM_V_at) READELF=$(READELF) CPPFILT=$(CPPFILT) python3 $(top_srcdir)/contrib/devtools/symbol-check.py < $(bin_PROGRAMS)
```

Comment:
nit: Don't we use `$(PYTHON)` for those?

Reply:
Good catch!